boards/shields/m5stack_m5dial/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+.. _m5stack_m5dial:
+
+M5Stack M5Dial
+##############
+
+Overview
+********
+
+M5Stack M5Dial is an ESP32-based development board from M5Stack.
+The device features an embedded M5Stack-StampS3 module that runs
```

Comment:
:) needs the global search & replace in the context of M5Dial too, I guess!
Also, it might make sense to make all mentions (or at least this one as this if the first one in the page) to StampS3 a link to the board page

```suggestion
The device features an embedded M5Stack StampS3 module that runs
```